### PR TITLE
[NET-954] Searchable reference section

### PR DESCRIPTION
**Pull request by johndgrey**
[[NET-954]](https://nethinks.atlassian.net/browse/NET-954)

[NET-954]: https://nethinks.atlassian.net/browse/NET-954

### CODE REVIEW

File: app/src/app/search/search.component.ts [hunk 1]
```diff
@@ -169,7 +146,7 @@ export class SearchComponent implements OnInit, OnDestroy {
             relativeTo: this.route,
             queryParams: { limit: this.limit },
             queryParamsHandling: 'merge'
-          });
+          }).then();
```
Comment (markheumueller) [inline app/src/app/search/search.component.ts:149]: Is the promise resolving necessary here?

File: app/src/app/search/search.component.ts [hunk 2]
```diff
@@ -318,12 +209,11 @@ export class SearchComponent implements OnInit, OnDestroy {
         relativeTo: this.route,
         queryParams: { page },
         queryParamsHandling: 'merge'
-      });
+      }).then();
```
Comment (markheumueller) [inline app/src/app/search/search.component.ts:212]: Same here?

File: cmdb/interface/rest_api/search_routes.py [hunk 3]
```diff
@@ -84,14 +84,10 @@ def search_framework(request_user: UserModel):
...
-                                                                       active=only_active, user=request_user,
-                                                                       permission=AccessControlPermission.READ)
-        else:
-            query: Pipeline = builder.build(search_parameters, object_manager,
-                                            user=request_user,
-                                            permission=AccessControlPermission.READ, active_flag=only_active)
+
+        query: Pipeline = builder.build(search_parameters, object_manager,
+                                        user=request_user,
+                                        permission=AccessControlPermission.READ, active_flag=only_active)
```
Comment (markheumueller) [inline cmdb/interface/rest_api/search_routes.py:90]: Wouldn't it be cleaner to set a resolution of the active flag before?
For me it is the same:
`active` -> `{"active": true}` in search_parameters

**Review by markheumueller — COMMENTED** [review]
That is the only thing I would like to note.

File: app/src/app/search/search.component.ts (hunk 1 above)
Comment (johndgrey) [reply app/src/app/search/search.component.ts:149]: No not really. Does my IDE automatically

**Review by markheumueller — APPROVED** [review]

**Pull request merged**